src/java/org/apache/cassandra/locator/PendingRangeMaps.java

```diff
@@ -23,196 +23,176 @@
 import com.google.common.collect.Iterators;
 import org.apache.cassandra.dht.Range;
 import org.apache.cassandra.dht.Token;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.util.*;
 
-public class PendingRangeMaps implements Iterable<Map.Entry<Range<Token>, List<InetAddressAndPort>>>
+public class PendingRangeMaps implements Iterable<Map.Entry<Range<Token>, List<Replica>>>
```

Comment:
Should this be consistent and stick to ReplicaList so we don't accidentally emit List<Replica> into the rest of Cassandra?

Reply:
fixed